resources/quality/cartesio/abs/cartesio_0.25_abs_high.inst.cfg

```diff
@@ -18,7 +18,6 @@ wall_thickness = 1
 top_bottom_thickness = 0.8
 wall_0_inset = -0.05
 fill_perimeter_gaps = nowhere
-travel_compensate_overlapping_walls_enabled =
```

Comment:
I wonder how these empty values ever got through. I'd even expect them to give a syntax error maybe.